src/style.css

```diff
@@ -1,3 +1,89 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts. For instance, if 'Roboto' fails to load, 'Arial' will be used, and if 'Arial' isn't available, the browser's default sans-serif font will be used.